src/Components/Quizz/Quizz.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './Quizz.css';
+import StarsQuizz from './StarsQuizz';
+import PropositionsQuizz from './PropositionsQuizz';
+
+
+
+class Quizz extends React.Component {
+    
+    state = {
+        question: null,
+        correct_answer: null,
+        incorrect_answers: null,
+   
+    }
+
+    getQuestions(){
+        fetch("https://opentdb.com/api.php?amount=20&category=11&difficulty=easy")
+        .then(res => res.json())
+        .then(res => this.setState({ ...res.results[0] }))
+    }
+
+    componentDidMount(){
+        this.getQuestions()
+    }
+
+    
+    
+    render() {
+        const {question, difficulty,correct_answer,incorrect_answers} = this.state;
+        
+        return (
+        
+            <div className='contentQuizz '>
+                <h1 className='titleQuizz text-focus-in'>Que la Force soit avec toi !!!</h1>
```

Comment:
Typographic rules discourage `!!!`, you should only use one at a time `!`. Otherwise cosmic entities from beyond the galaxy will try to kill you.